### ISSUE TITLE
Rayleigh_Ray2, PnPn-2 failure

### ISSUE THREAD

**Issue by RonRahaman** [Nek5000/NekExamples#58]
The Rayleigh_Ray2, PnPn-2 test fails with the following error:
```
1:     FAILURE: umax: Test value 0.00662153 exceeds target value 0.006728787 +/- 1e-05
```

Nek5000/Nek5000@e6462392c8ee7dca3497eb23110b74de85413e02 is the first bad commit

**Comment by stgeke** [comment]
I think the delta of 1e-5 is too tight for regression testing. Even small variations (e.g. disabling residual projection, different machine) produce a slight different result in order of 1e-5. I get same test value of `6.621518E-03` using the current master and the latest svn version.

I recommend to set delta to 1e-4.

**Comment by stgeke** [comment]
I can reproduce the test value of `0.006728787`with r1073 on my machine. The latest SVN version (r1142) produces `6.621518E-03`. 

Note, this cases uses estimated varying residual for the pressure and Helmholtz solve. If I use fixed residual tolerances I get the same test value for both revisions. 

Instead of relaxing delta we could also set fixed residual tolerances.

**Comment by RonRahaman** [comment]
Fixed in PR #66
